code/modules/mob/living/brain/brain_item.dm

```diff
@@ -39,6 +39,18 @@
 	/// Maximum skillchip slots available. Do not reference this var directly and instead call get_max_skillchip_slots()
 	var/max_skillchip_slots = 5
 
+	/// Size modifier for the sprite
+	var/brain_size = 1
+
+/obj/item/organ/internal/brain/Initialize(mapload)
+	. = ..()
+	// Brain size logic
+	transform = transform.Scale(brain_size)
+	if(brain_size < 1)
+		desc += "\nIt is a bit on the smaller side"
+	if(brain_size > 1)
+		desc += "\nIt is bigger than average"
```

Comment:
```suggestion
	switch(brain_size)
		if(0 to 1)
			desc += span_info("\nIt is a bit on the smaller side...")
		if(1 to INFINITY)
			desc += span_info("\nIt is bigger than average...")
```

Reply:
actually just move all of this to `update_desc()` and then invoke `update_appearances()` (or whatever the proc+flag combo is) in `Initialize()`

Reply:
Sorry @san7890, you have failed your surprise coding audit.
```
	switch(brain_size)
		if(0 to 1)
			desc += "\nIt is a bit on the smaller side..."
		if(1 to INFINITY)
			desc += "\nIt is bigger than average..."
```

Will result in brains with a size of 1 being marked as having the added description of "\nIt is a bit on the smaller side..." on account of the fact that the number 1 is included in the range 0 to 1.

Reply:
ah right we don't want to append a description if the brain size is exactly `1`, my bad. i still think this should all be in update_desc anyways for when an admin does stuff to it